sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -625,7 +625,7 @@ case class JsonToStructs(
        {"a":1,"b":2}
       > SELECT _FUNC_(named_struct('time', to_timestamp('2015-08-26', 'yyyy-MM-dd')), map('timestampFormat', 'dd/MM/yyyy'));
        {"time":"26/08/2015"}
-      > SELECT _FUNC_(array(named_struct('a', 1, 'b', 2));
+      > SELECT _FUNC_(array(named_struct('a', 1, 'b', 2)));
```

Comment:
This typo was introduced at [the beginning](https://github.com/dongjoon-hyun/spark/commit/0cdcf9114527a2c359c25e46fd6556b3855bfb28#diff-6626026091295ad8c0dfb66ecbcd04b1R605). Could you review this, @HyukjinKwon ?